### PR TITLE
Docs to make theme command launch with content

### PR DESCRIPTION
**Pull request by pvpg**
- Added documents for the theme command
- Changed composer.json details that caused a warning
- Changed the README
- Added prismic-configuration.json file

### CODE REVIEW

File: custom_types/menu.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"Main":{"title":{"type":"StructuredText","config":{"placeholder":"Menu title...","single":"heading1"}},"menu_links":{"type":"Group","config":{"fields":{"label":{"type":"StructuredText","config":{"single":"paragraph","label":"Link Label","placeholder":"Link Label..."}},"link":{"type":"Link","config":{"label":"Link","placeholder":"Select a Link..."}}},"label":"Menu Links"}}}}
```
Comment (levimykel) [inline custom_types/menu.json]: Beautify this file

File: custom_types/page.json [hunk 2]
```diff
@@ -0,0 +1 @@
...Link","placeholder":"Optional Link"}},"link_label":{"type":"StructuredText","config":{"single":"paragraph","label":"Link Label","placeholder":"Optional Link Label","isEverythingAuthorized":false,"authorized":["paragraph","text"],"isAuthorized":{}}},"image":{"type":"Image","config":{"constraint":{"width":727,"height":402},"thumbnails":[{"name":"thumbnail","width":400,"height":null}],"label":"Image"}}}},"image_highlight":{"type":"Slice","fieldset":"Image Highlight","description":"A section to highlight an image with title, text, and optional link","icon":"star","non-repeat":{"featured_image":{"type":"Image","config":{"constraint":{"width":727,"height":402},"thumbnails":[],"label":"Featured Image"}},"title":{"type":"StructuredText","config":{"single":"heading2","label":"Image Title","placeholder":"Featured Title..."}},"headline":{"type":"StructuredText","config":{"single":"heading3","label":"Image Headline","placeholder":"Featured Headline..."}},"link":{"type":"Link","config":{"label":"Featured Link","placeholder":"Optional Link"}},"link_label":{"type":"StructuredText","config":{"single":"paragraph","label":"Featured Link Label","placeholder":"Optional Link Label"}}},"repeat":{}}}}}}}
```
Comment (levimykel) [inline custom_types/page.json]: Beautify this file

File: prismic-configuration.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "apiEndpoint": "https://your-repo-name.cdn.prismic.io/api/v2"
```
Comment (levimykel) [inline prismic-configuration.json]: What is the purpose of this?

Comment (pvpg) [reply prismic-configuration.json]: deleted!

**Pull request merged**